lib/babe/babe_integration_test.go

```diff
@@ -54,26 +54,14 @@ var (
 
 //go:generate mockgen -destination=mock_telemetry_test.go -package $GOPACKAGE github.com/ChainSafe/gossamer/dot/telemetry Client
 
-func createTestService(t *testing.T, cfg *ServiceConfig) *Service {
+func createTestService(t *testing.T, cfg ServiceConfig) *Service {
 	wasmer.DefaultTestLogLvl = 1
 
 	gen, genTrie, genHeader := genesis.NewDevGenesisWithTrieAndHeader(t)
 	genesisHeader = genHeader
 
 	var err error
 
-	if cfg == nil {
-		cfg = &ServiceConfig{
-			Authority: true,
-		}
-	}
-
-	cfg.BlockImportHandler = new(mocks.BlockImportHandler)
-	cfg.BlockImportHandler.(*mocks.BlockImportHandler).
-		On("HandleBlockProduced",
-			mock.AnythingOfType("*types.Block"), mock.AnythingOfType("*storage.TrieState")).
-		Return(nil)
-
```

Comment:
note this is pushed back up to the caller (tests), since it was differing from one test to the other.